CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # Change Log
 
+<<<<<<< HEAD
```

Comment:
I know, but I don't know how to resolve it without reverting changes and starting over. The `develop` branch was fetched after the fix. I still had to use `git merge -s recursive -Xtheirs upstream/develop` to fix the local conflicts. I never touched those files locally.